eth_tester/utils/backend_testing.py

```diff
@@ -264,8 +264,10 @@ def test_mine_multiple_blocks(self, eth_tester):
     #
     @pytest.mark.parametrize('is_pending', [True, False])
     def test_send_raw_transaction_valid_raw_transaction(self, eth_tester, is_pending):
-        if is_pending and not hasattr(eth_tester.backend, 'send_signed_transaction'):
-            pytest.xfail("backend does not support presigned pending transactions")
+        if is_pending:
+            from eth_tester.backends import PyEthereum16Backend, PyEthereum21Backend
```

Comment:
👍 

I think we're right around the corner from being able to deprecate those backends.  They were originally written to provide a smooth upgrade path for populus, but it's not looking like that is actually going to happen (needing a smooth upgrade path) since populus is looking like it's going to require some significant gutting to get back to a nice state at which point forcing people onto the new py-evm based backend is not really a big deal anymore.

Nothing actionable other than going forward we should probably just accept failures in the pyethereum backends and mark them as xfails or something.